src/db.py

```diff
@@ -14,7 +14,8 @@
 STATUS_RECEIPT_UNKNOWN_USER = 1
 STATUS_RECEIPT_ALREADY_EXIST = 2
 STATUS_USER_ALREADY_EXIST = 3
-STATUS_MAIL_ALREADY_EXIST = 4
+STATUS_USER_ALREADY_DEACTIVATED = 4
```

Comment:
It's not a big deal here, but might cause a lot of problems somewhere. It had better do not change codes.